### ISSUE TITLE
Unexpected output from model.eval

### ISSUE THREAD

**Issue by Betaglutamate** [MouseLand/cellpose#544]
Hi,

I am not sure if I am making a mistake somewhere or misunderstanding the code, so please correct me if I am.

When running:

`imgs = cellpose.io.imread([list of images])`  In this case just a single image

`masks, flows, styles = model.eval(imgs, diameter=None, channels=channels, compute_masks=True)`

The returned output for masks is a single multi label image of masks (see attached)
I expected a list of masks instead, with each list item corresponding to one cell in the image.

![test](https://user-images.githubusercontent.com/39518170/183404838-d829c25a-f041-4f42-91ac-a491fbad3ca6.jpg)

**Comment by jonasutz** [comment]
This is indeed the expected output. 
But you could easily convert this into a list of binary masks. In the case of a singular image:
`list_of_masks = [masks == c for c in range(1,masks.max()+1)]`

**Comment by Betaglutamate** [comment]
Ok great that solves my issue. I was using a very complicated way to extract the masks by saving the image and then loading it back into my program then extracting all the unique colours in the image then I would check if image == unique colour and that would give me each mask which made the program very slow. Your way is way more efficient thank you.
